fedocal/templates/default/master.html

```diff
@@ -114,6 +114,19 @@ <h4 class="hide">{{ _('Main Site Links') }}</h4>
             });
         });
     </script>
+
+    {% if config['FEDMENU_URL'] is defined %}
```

Comment:
How does ``is defined`` work here?  I thought ``is defined`` was a jinja test to see if a var was defined.. but, ``config['FEDMENU_URL']`` is more than just a simple var.